### DISCUSSION TITLE
go做后端用cherry-markdown开发桌面客户端，有没有搞头？

### DISCUSSION THREAD

**Discussion by zimolab** [Tencent/cherry-markdown#378] [Show and tell]
rt
目前实现了一些功能，还很初步，和electron不同的一种技术路线。
[![pSewwKx.png](https://s1.ax1x.com/2023/01/09/pSewwKx.png)](https://imgse.com/i/pSewwKx)

**Comment by sunsonliu** [comment]
似乎可以尝试，不过目前大家可能更倾向用electron，毕竟在客户端方面electron的生态和成功案例都是非常丰富的
